Add link in docs menu to stable docs

zou3519 opened the pull request:
Part of #5738. Warns users that they're not viewing the latest stable
release docs.

We should remember to delete this when cutting out 0.4.0 release docs. (we'd just delete the div in pytorch.github.io)

This is what it looks like now:
![image](https://user-images.githubusercontent.com/5652049/38576477-47046124-3ccc-11e8-83e3-767d2d61948b.png)

cc @SsnL @soumith

apaszke reviewed (changes requested):
It looks like you included some accidental changes in TH

zou3519 commented:
Oops. Thanks for the catch!

soumith commented:
you caught @zou3519 on his clever plan to take over the world...

zou3519 committed:
Add link in docs menu to stable docs

Part of #5738. Warns users that they're not viewing the latest stable
release docs.

We should remember to delete this when cutting out 0.4.0 release docs. (we'd just delete the div in pytorch.github.io)

apaszke commented:
No sneaky changes will get in under my watch 😎
